code/datums/components/slippery.dm

```diff
@@ -21,5 +21,7 @@
 
 
 /datum/component/slippery/proc/Slip_on_wearer(datum/source, atom/movable/AM, mob/living/crossed)
+	var/obj/item/I = crossed.get_item_by_slot(ITEM_SLOT_FEET)
 	if(!(crossed.mobility_flags & MOBILITY_STAND) && !crossed.buckle_lying)
-		Slip(source, AM)
+		if (istype(I, /obj/item/clothing/shoes/clown_shoes))
+			Slip(source, AM)

```

Comment:
Changing the slippery component will affect everything that uses it, which is more than clown items.

Consider doing further research before making changes like these. For example:
 1) When is `Slip_on_wearer` called?
     - It is registered to the `COMSIG_ITEM_WEARERCROSSED` signal
 2) What triggers `COMSIG_ITEM_WEARERCROSSED`?
     - `/mob/living/Crossed` signals it on all worn items
 3) Which item do clowns wear that has `/datum/component/slippery`?
     - `/obj/item/pda/clown`
 4) How can I modify `/obj/item/pda/clown`'s behavior to do what I want without affecting the other items that use `/datum/component/slippery`?
     - Make a `/datum/component/slippery` subtype with this behavior
     - Replace `/obj/item/pda/clown`'s slippery component with the subtype

Reply:
Possible further research

5) Since the clown's PDA is responsible for slipping people, does my proposed change still make sense?
     - ???

Reply:
Yep probably for the best, I'll do it tomorrow bright and early. But do enlighten me if you found any other slippery wearables as I could not find any and the hands don't count for COMSIG_ITEM_WEARERCROSSED.